File: MPI/mpicheck/tests/functions/2.0/MPI_Type_set_attr.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER(KIND=MPI_ADDRESS_KIND) var_2
       INTEGER var_3
        call mpi_type_set_attr(var_0, var_1, var_2, var_3)
       call pmpi_type_set_attr(var_0, var_1, var_2, var_3)
        end program main
    

File: MPI/mpicheck/tests/functions/2.0/MPI_Type_set_attr.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Datatype var_0;
    int var_1;
    void *var_2;
    int ret;
    /* calls */
    ret = MPI_Type_set_attr(var_0, var_1, var_2);
    ret = PMPI_Type_set_attr(var_0, var_1, var_2);
    return 0;
}